src/util/planHelper.ts

```diff
@@ -313,4 +313,14 @@ export const dateFormat = (timestamp: number, showTime = false) => {
   return resultString;
 };
 
+export const textMonthDateFormat = (timestamp: number): string => {
+  const date = new Date(timestamp * 1000);
+
+  const month = date.toLocaleString(undefined, { month: 'long' });
```

Comment:
why `undefined`? we should format this according to the user's locale

Reply:
And that's what `undefined` does here, the user's locale is taken.

"If you omit this parameter, the default locale of the JavaScript runtime is used" - from the docs